### DISCUSSION TITLE
WOW and thank you!

### DISCUSSION THREAD

**Discussion by thisisjoshjones** [Akifyss/obsidian-border#131] [General]
Just want to say this theme has me falling head over heels, Obsidian looks so fresh now and the auto-hides are game changers. Thank you so much!

<img width="1470" alt="Screenshot 2023-11-29 at 7 35 19 PM" src="https://github.com/Akifyss/obsidian-border/assets/130625079/a891ef02-f019-4f61-8069-f312ff2764ee">

**Comment by Akifyss** [comment]
Glad to hear you're loving the theme!❤️
